src/analysis/special_functions/integrals.lean

```diff
@@ -226,6 +227,12 @@ end
 lemma integral_cos : ∫ x in a..b, cos x = sin b - sin a :=
 by rw integral_deriv_eq_sub'; norm_num [continuous_on_cos]
 
+@[simp]
+lemma integral_cos_sq_sub_sin_sq :
+  ∫ x in a..b, cos x ^ 2 - sin x ^ 2 = sin b * cos b - sin a * cos a :=
+by simpa only [pow_two, sub_eq_add_neg, neg_mul_eq_mul_neg] using integral_deriv_mul_eq_sub
+  (λ x hx, has_deriv_at_sin x) (λ x hx, has_deriv_at_cos x) continuous_on_cos continuous_on_sin.neg
```

Comment:
I'm not sure that this integral is important enough to have a dedicated `simp` lemma. What about lemmas for `cos x ^ 2` and `sin x ^ 2`?

Reply:
Yes, I can add those, but I think I will need to add some more API first in order to be able to do them efficiently. Should I remove the `simp` from here and create a new PR with `cos x ^ 2` and `sin x ^ 2`?

Reply:
@benjamindavidson I think removing the simp would be good (at least for now)

Reply:
> Yes, I can add those, but I think I will need to add some more API first in order to be able to do them efficiently. Should I remove the `simp` from here and create a new PR with `cos x ^ 2` and `sin x ^ 2`?

Ref. #7065

Reply:
Done.